index.bs

```diff
@@ -769,13 +768,16 @@ precomposed character).</p>
 <p>The WebVTT file is a container file for chunks of data that are time-aligned with a video or
 audio resource. It can therefore be regarded as a serialisation format for time-aligned data.</p>
 
-<p>A WebVTT file starts with a header and then contains a series data blocks. If a data block has a
-start and end time, it is called a WebVTT cue. A comment is another kind of data block.</p>
+<p>A WebVTT file starts with a header and then contains a series of data blocks. If a data block has
+a start and end time, it is called a WebVTT cue. A comment is another kind of data block.</p>
 
-<p>A WebVTT file can contain cues with different kinds of data. The HTML specification identifies
-captions, subtitles, chapters, audio descriptions and metadata as data kinds.</p>
+<p>A WebVTT files can contain different kinds of data. The HTML specification identifies captions,
```

Comment:
The indefinite article with a plural doesn't read well. Actually this sentence needs reformulating to say what it means, because it's hard to avoid giving the impression that one file might be able to contain multiple kinds of data. Maybe something like:

_WebVTT files can be used to express different kinds of data; any one WebVTT file can contain one such kind._

or, since the prevention is mentioned below:

_Different kinds of data can be carried in WebVTT files._

Reply:
Thanks.